CMakeLists.txt

```diff
@@ -262,7 +262,7 @@ ENDIF()
 
 MY_CHECK_AND_SET_COMPILER_FLAG("-Wno-deprecated-copy" DEBUG RELEASE RELWITHDEBINFO MINSIZEREL)
 MY_CHECK_AND_SET_COMPILER_FLAG("-Wno-deprecated-declarations" DEBUG RELEASE RELWITHDEBINFO MINSIZEREL)
-MY_CHECK_AND_SET_COMPILER_FLAG("-Werror -Wall -Wextra")
```

Comment:
You should get this back.

Reply:
Yes, in a minute.

It was necessary because ASAN produces a lot of warnings about boost-related things and local build does not build.